gemfirexd/tools/src/dunit/java/com/pivotal/gemfirexd/ClientServerDUnit.java

```diff
@@ -1615,10 +1615,11 @@ public void run() {
    * Test if multiple connections from network clients failover successfully.
    */
   public void testNetworkClientFailover() throws Exception {
-    // start some servers not using locator
-    startVMs(0, 3);
-    // Start a network server
-    final int netPort = startNetworkServer(1, null, null);
+    // start some servers
+    startVMs(1, 3);
```

Comment:
Also why are we starting a client now?

Reply:
Without the client, the "ClientSharedUtils.isThriftDefault()" property is not set correctly (to false for the GemXD tests) because node has not booted up. This causes the obtained URL to be like "jdbc:gemfirexd:drda://". Adding a client allows the check to be normal one and mimic rowstore mode better without requiring explicit filtering of ":drda:" which is an internal detail.